### DISCUSSION TITLE
External links

### DISCUSSION THREAD

**Discussion by MemberGC** [google/docsy#1750] [Q&A]
![Screenshot_20231119-085144_Chrome.jpg](https://github.com/google/docsy/assets/86789687/d6ddbb1e-f935-417c-9ebe-d4ec24f4bf60)

Hey I want to add external links in my docus.dev markdown in side bar how do I do it.
Like given in the image.

**Comment by fekete-robert** [comment]
You mean something like this? [https://www.docsy.dev/docs/adding-content/navigation/#add-manual-links-to-the-section-menu](https://www.docsy.dev/docs/adding-content/navigation/#add-manual-links-to-the-section-menu)

**Reply by MemberGC** [reply]
I think so. 
I am new to github and to make docs using markdown too.
So in navigation bar where I can add external links.
